dev/settings.gradle

```diff
@@ -187,6 +189,9 @@ if (!JavaVersion.current().isJava11Compatible()) {
 
 }
 
+//DirectoryScanner.removeDefaultExclude("**/.gitignore")
```

Comment:
Are these being left in for future use? If they are, should a comment be left for them? If not, should these lines be removed?

Reply:
Sorry, I don't know the answer to this. This is a change that Ethan had done.